content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+4. The **Gradient Transform Modifier** component displays a warning about missing a required component. Choose **Add Required Component** and select **Shape Reference** from the list. In a later step, you'll add a reference to the **Axis Aligned Box Shape** component in the Terrain Spawner entity to ensure the terrain bounds and the gradient bounds are identical.
+
+5. Add a heightmap to the Terrain Gradient entity. In the **Image Gradient** component, to the right of the **Image Asset** property, click the {{< icon "file-folder.svg" >}} **File** button and select your heightmap asset.
+
+    {{< tip >}}
+Changing the **Sampling Type** property in the **Image Gradient** component from `Point` to `Bilinear` might make the elevation changes smoother if the heightmap resolution and the dimensions of the **Axis Aligned Box Shape** component differ.
+    {{< /tip >}}
+
+6. The **Shape Reference** component needs to reference the **Axis Aligned Box Shape** component in the Terrain Spawner entity. In the **Shape Reference** component, click the {{< icon "picker.svg" >}} **Picker** button, then click the Terrain Spawner entity in Entity Outliner to create the reference. The following image shows the final Terrain Gradient entity:
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-gradient-entity.png" width="450" alt="The completed Terrain Gradient entity." >}}
+
+7. Select the Terrain Spawner entity. In the **Terrain Height Gradient List** component, click the {{< icon "add.svg" >}} **Add** button to add a new gradient slot. Click the {{< icon "picker.svg" >}} **Picker** button, then click the Terrain Gradient entity in Entity Outliner to create a reference.
+
+8. **Save** the level (hotkey **Ctrl+S**).
+
+{{< note >}}
+Before continuing to the next section, be sure to learn about terrain setup with Landscape Canvas by selecting the tab at the top of this section.
+{{< /note >}}
+
+{{% /tab %}}
+
+{{% tab name="Terrain setup with Landscape Canvas" %}}
+
+If you worked through terrain setup with entities in the left tab, you can see how terrain creation might be time consuming and difficult to troubleshoot. Landscape Canvas can greatly simplify the process and provide an easy to understand visual graph for the relationships between the various entities that generate the terrain. Landscape Canvas is particularly useful when you create complex networks of gradients and gradient modifiers to define terrain elevation.
+
+In this section, you'll create terrain based on a heightmap with Landscape Canvas.
+
+1. If there is an existing terrain in your level, delete or hide the terrain entities.
```

Comment:
Deleting is the right thing to do here. Hiding will cause them not to be shown in the Editor, but they will still appear and get used in game mode or the launcher. Might also want to clarify that the terrain level entities are still needed, so the step for adding them should appear here too.

Reply:
I take it for granted users are just working on the content in the tutorial, maybe I shouldn't. Fixed.